### PR TITLE
Bug 1986504: update docs on tokens

### PR DESCRIPTION
**Pull request by slaskawi**
https://bugzilla.redhat.com/show_bug.cgi?id=1986504

### CODE REVIEW

File: oauth/v1/types.go [hunk 1]
```diff
@@ -8,7 +8,9 @@ import (
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// OAuthAccessToken describes an OAuth access token
+// OAuthAccessToken describes an OAuth access token.
+// The name of the token objects is the sha256 hash of the actual token (not decoded, but plain base64) encoded as base64
```
[the remark this answers is not in the document]
Comment (slaskawi) [reply oauth/v1/types.go]: I added small clarification. Please check if you're fine with it.

File: oauth/v1/types.go [hunk 2]
```diff
@@ -8,7 +8,10 @@ import (
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// OAuthAccessToken describes an OAuth access token
+// OAuthAccessToken describes an OAuth access token.
+// The name of the token object is a sha256 hash of the actual token (not decoded, but plain base64) encoded as base64
+// prefixed with "sha256~" string. It is highly recommended to use base64 with URL and Filename Safe Alphabet (RFC4648)
+// for encoding.
```
Comment (sttts) [inline oauth/v1/types.go:14]: Not sure what `It is highly recommended to use base64 with URL and Filename Safe Alphabet (RFC4648) for encoding." means.

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**